### PR TITLE
New port: security/pass-otp

### PR DESCRIPTION
**Pull request by lpefferkorn**
###### Type(s)

- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on

macOS 10.11.6 15G22010
Xcode 8.x

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?

- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

File: security/pass-otp/Portfile [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8::et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        tadfisher pass-otp 1.1.1 v
+revision            1
```
Comment (yan12125) [inline security/pass-otp/Portfile]: In general `revision` is 0 for a new version, and 0 is the default, so the revision line is usually omitted.

File: security/pass-otp/Portfile [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+maintainers         loicp.eu:loic-github
```
Comment (yan12125) [inline security/pass-otp/Portfile]: Please also add your github handle with the following format:
```
{loicp.eu:loic-github @lpefferkorn}
```

File: security/pass-otp/Portfile (hunk 1 above)
Comment (lpefferkorn) [reply security/pass-otp/Portfile]: Ok, I'll drop this line then

File: security/pass-otp/Portfile (hunk 2 above)
Comment (lpefferkorn) [reply security/pass-otp/Portfile]: Sure!

**Comment by lpefferkorn** [comment]
Fixed!

File: security/pass-otp/Portfile [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8::et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        tadfisher pass-otp 1.1.1 v
+maintainers         {loicp.eu:loic-github @lpefferkorn}
```
Comment (mf2k) [inline security/pass-otp/Portfile]: Can openmaintainer be added?

File: security/pass-otp/Portfile [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+categories          security
+description         A pass extension for managing one-time-password (OTP) tokens
+long_description    ${description}
```
Comment (mf2k) [inline security/pass-otp/Portfile]: We prefer to not simply copy the short description. It's not a blocker, but can the `long_description` be expanded?

**Comment by mf2k** [comment]
The first line of the commit message should be changed to the following:

`pass-otp: New port`

File: security/pass-otp/Portfile (hunk 3 above)
Comment (pmetzger) [reply security/pass-otp/Portfile]: We indeed generally prefer that ports be `openmaintainer` unless there's good reason not to do that.

**Comment by pmetzger** [comment]
@lpefferkorn Thanks for the contribution! Once @mf2k's concerns are addressed, we should be able to merge this quickly.

File: security/pass-otp/Portfile (hunk 3 above)
Comment (lpefferkorn) [reply security/pass-otp/Portfile]: Will add!

**Comment by lpefferkorn** [comment]
Regarding  **long_description**, I cannot find something that would not paraphrase the short one and add something useful to it.

**Pull request merged**